.github/workflows/test.yml

```diff
@@ -59,33 +72,14 @@ jobs:
 
     - name: install dependencies
       run: |
-        pip install .  # Install from setup.py
-        pip install -q pytest
+        # pip install .  # Install from setup.py
+        # pip install -q pytest
+        # python -m pip install --upgrade pip
+        pip install tox tox-gh-actions
 
     - name: test
       run: |
         echo $PATH
         which nvim
         nvim --version
-        python -m pytest
-
-    - uses: codecov/codecov-action@v3
-      if: runner.os == 'macOS'
-      with:
-        verbose: true # optional (default = false)
```

Comment:
I gave up on codecov for now. https://github.com/neovim/pynvim/pull/469/commits/68bde2069b58b369a2a87ecdade8e77d28dbe68d

```
    ERROR: Failed building wheel for pyuv
    ERROR: Could not build wheels for pyuv, which is required to install pyproject.toml-based projects
```